config/routes.rb

```diff
@@ -74,17 +75,13 @@
         end
       end
 
-      resources :criteria do
+      resources :criteria, except: [:index] do
         collection do
           post 'update_positions'
         end
       end
 
-      resources :rubric_criteria do
-        member do
-          delete 'destroy'
-        end
-
+      resources :rubric_criteria, only: [:index] do
```

Comment:
I can double check, but we have an index action in the rubric and flexible controllers, so it seems  reasonable to have a route to display the marking scheme page.